teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization:, **parameters)
+    super(**parameters)
```

Comment:
- [ ] I was unable to create a teacher as well. I would recommend you fix this based on the previous suggestion for the Student class.